src/app/project/project-details-tab/project-details-tab.component.ts

```diff
@@ -44,13 +47,17 @@ export class ProjectDetailsTabComponent implements OnInit, AfterViewInit, OnDest
     this.commentPeriod = this.project.commentPeriodForBanner;
     this.route.data.subscribe((res: any) => {
       if (res) {
-        if (res.shapefile && res.shapefile[0].data.meta && res.shapefile[0].data.meta.length > 0) {
-          this.shapefile = res.shapefile[0].data.searchResults;
+        if (res.documents && res.documents[0].data.meta && res.documents[0].data.meta.length > 0) {
+          this.shapefile = res.documents[0].data.searchResults;
         } else {
           this.shapefile = [];
         }
       }
     });
+    // The following items are loaded by a file that is only present on cluster builds.
+    // Locally, this will be empty and local defaults will be used.
+    const remote_api_path = window.localStorage.getItem('from_public_server--remote_api_base_path');
```

Comment:
This is the WordPress php in you coming out try `const remoteApiPath =`

Reply:
Fixed in latest push